tools/release/README.md

```diff
@@ -35,30 +35,45 @@ the new image as described below:
 
 ### Manual Release Process
 
-Prior to running the `publish.sh` script, we have to download the main image
-that will be released:
-
-    docker run -it -p "127.0.0.1:8081:8080" -v "${HOME}:/content" \
-      -e "PROJECT_ID=<PROJECT_ID>" \
-      "gcr.io/cloud-datalab/datalab:local-$(date +%Y%m%d)"
-
-... and then run the following notebooks to make sure all of the graphs and
-charts are displayed properly: 
-
-    samples/Conversion Analysis with Google AnalyticsData.ipynb 
-    tutorials/Data/Interactive Charts with Google Charting APIs.ipynb 
+1. Bump up the hard-coded Datalab version to the next semantic version in the
+[tools/release/version.sh](https://github.com/googledatalab/datalab/blob/master/tools/release/version.sh)
+file, send a PR with the changes and merge.
+
+2. Try the new image before releasing it. You can either try it locally:
+```
```

Comment:
Remove the bit about trying it locally. We should always run `datalab create` with the new image before releasing it.